lib/models/note.js

```diff
@@ -42,9 +42,13 @@ module.exports = function (sequelize, DataTypes) {
       type: DataTypes.STRING,
       unique: true
     },
-    permission: {
+    viewableBy: {
       type: DataTypes.ENUM,
-      values: permissionTypes
+      values: userTypes
```

Comment:
Careful with this change, it'll require a database migration to be handled properly for existing setups